package.json

```diff
@@ -11,73 +11,59 @@
     "start": "npm run clean && npm run build && node dist/src/server.js",
     "build": "npm run clean && tsc -b",
     "clean": "tsc -b --clean",
-    "test": "TS_NODE_FILES=true tap --test-ignore='^dist'",
+    "test": "TS_NODE_FILES=true tap --ts --no-check-coverage --test-ignore='^(dist|scripts)'",
```

Comment:
`--no-check-coverage` still shows the coverage matrix, but will avoid logging errors that complain about not reaching the minimum test coverage percentage